### PR TITLE
QMAPS-1914 change CSS in itinerary list

### PR DESCRIPTION
**Pull request by xem**
## Description
Change display of details and close buttons on mobile in the itinerary list.
And various fixes to match the design better.

## Screenshots
image

### CODE REVIEW

File: src/panel/direction/RouteSummary.jsx [hunk 1]
```diff
@@ -37,6 +37,7 @@ export default class RouteSummary extends React.Component {
     return <Flex
       className="itinerary_leg_summary"
       justifyContent="space-between"
+      alignItems="flex-end"
```
Comment (bbecquet) [inline src/panel/direction/RouteSummary.jsx:40]: This results in a proptype console warning, as this value isn't explicitely declared yet in our `<Flex>` component.

File: src/scss/includes/direction-panel.scss [hunk 2]
```diff
@@ -54,13 +54,16 @@
...
-.direction-panel-mobile {
   &.maximized, &.panel--holding {
     .itemList > div:not(:first-child) {
       display: block;
+      margin-top: 12px;
     }
   }
+
+  .panel-close {
+    top: 15px;
```
Comment (bbecquet) [inline src/scss/includes/direction-panel.scss:66]: This isn't taken into account as the selector is less specific than the existing one.

File: src/scss/includes/direction.scss [hunk 3]
```diff
@@ -76,11 +78,12 @@
 }
 
 .itinerary_leg_summary {
-  padding: $spacing-l;
+  padding: 0 $spacing-s;
```
Comment (bbecquet) [inline src/scss/includes/direction.scss:81]: Why this change? As a result, the button is too close to the bottom of the screen and two consecutive summaries are too close when the panel is deployed.

Capture d’écran de 2021-01-15 10-13-10

Capture d’écran de 2021-01-15 10-13-18

**Comment by G-Ray** [comment]
Looks good to me

**Review by G-Ray — APPROVED** [review]
